src/main/java/gregtech/common/pipelike/cable/net/EnergyNetHandler.java

```diff
@@ -71,18 +67,19 @@ public long acceptEnergyFromNetwork(EnumFacing side, long voltage, long amperage
                     }
```

Comment:
This is burning all the cables in the entire path, regardless of their max voltage.
Since this is so niche I wouldn't mind this as a "feature".

Reply:
Dan wanted it to be like this